src/seedu/addressbook/data/address/info/Block.java

```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.data.address.info;
+
+public class Block {
+	private String block;
+	
+	public Block(){
+		block = null;
+	}
+	
+	public void addInfo(String block){
+		this.block = block;
+	}
```

Comment:
The setting of this variable could be done in the constructor, reducing the need to call both the constructor and this setter.